suite-native/assets/src/components/EjectedDuringDiscoveryWarning.tsx

```diff
@@ -0,0 +1,33 @@
+import Animated from 'react-native-reanimated';
+import { useSelector } from 'react-redux';
+
+import { selectSelectedDevice } from '@suite-common/wallet-core';
+import { Box, InlineAlertBox } from '@suite-native/atoms';
+import { DeviceItemIcon } from '@suite-native/device';
+import {
+    SettingsSliceRootState,
+    selectHasDeviceBeenEjectedDuringDiscovery,
+} from '@suite-native/settings';
+
+export const EjectedDuringDiscoveryWarning = () => {
+    const device = useSelector(selectSelectedDevice);
+    const hasBeenEjectedDuringDiscovery = useSelector((state: SettingsSliceRootState) =>
+        selectHasDeviceBeenEjectedDuringDiscovery(state, device?.id),
+    );
+
```

Comment:
One of the conditions here should be if the device is connected, I guess, since I can see the warning to Reconnect your trezor even though the trezor is connected.